server/graphql/v2/interface/Account.js

```diff
@@ -322,6 +323,24 @@ const accountFieldsDefinition = () => ({
       return { nodes: result.rows, totalCount: result.count, limit, offset };
     },
   },
+  latestChangelogPublishDate: {
+    type: GraphQLDateTime,
+    async resolve(collective) {
+      const latestChangelogPublishDate = await models.Update.findAll({
+        where: {
+          CollectiveId: collective.id,
+          isChangelog: true,
+        },
+        attributes: [[sequelize.fn('max', sequelize.col('publishedAt')), 'date']],
+        raw: true,
+      });
+
+      if (latestChangelogPublishDate && !latestChangelogPublishDate.isEmpty) {
```

Comment:
Is this `isEmpty` provided by sequelize?

Reply:
@Betree : Ops, my mistake. This should be, `if (latestChangelogPublishDate && latestChangelogPublishDate.length > 0) {` 😮